model/src/main/java/oracle/kubernetes/weblogic/domain/v2/BaseConfiguration.java

```diff
@@ -1,17 +1,18 @@
-// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
 // Licensed under the Universal Permissive License v 1.0 as shown at
 // http://oss.oracle.com/licenses/upl.
 
 package oracle.kubernetes.weblogic.domain.v2;
 
+import com.google.gson.annotations.Expose;
```

Comment:
is this a new dependency? do we have a PLS record for GSON?

Reply:
No, this was in 1.0.  We get this transitively through the kubernetes-client.